pkgs/development/libraries/glib/default.nix

```diff
@@ -68,6 +68,7 @@ stdenv.mkDerivation (finalAttrs: {
   ] ++ lib.optionals stdenv.hostPlatform.isMusl [
     ./quark_init_on_demand.patch
     ./gobject_init_on_demand.patch
+    ./glib-tests-cxx-sentinel.patch
```

Comment:
Could you add a comment here pointing to the upstream MR that added the change?  Otherwise it requires quite a bit of digging to see where the patch comes from.

Reply:
Done